shell/platform/linux/fl_renderer.h

```diff
@@ -82,13 +82,13 @@ void fl_renderer_set_engine(FlRenderer* renderer, FlEngine* engine);
  * fl_renderer_add_view:
  * @renderer: an #FlRenderer.
  * @view_id: the ID of the view.
- * @view: the view Flutter is renderering to.
+ * @renderable: object that is to be rendered on.
  *
  * Add a view to render on.
  */
 void fl_renderer_add_view(FlRenderer* renderer,
```

Comment:
Is this a public method the end user can access in their runner app? If not, should we rename this to `fl_renderer_add_rendererable`?

Reply:
FlRenderer is private. I too was unsure if this should be renamed, however in the end I decided that since the it was a view ID it should be the view you are registering. I was also considering if FlRenderable should be FlSomethingView but I couldn't come up with a good name. Since it's all private I figured it probably wasn't worth worrying too much about.

Reply:
Nah, you're right. I've changed it.